docs/en/setup/backend/dynamic-config.md

```diff
@@ -56,6 +56,10 @@ For example:
 ```
 Supported configurations are as follows:
 
+| Config Key | SubItem Key Description |  Value Description | Value Format Example |
+|:----:|:----:|:----:|:----:|
+|core.default.endpoint-name-grouping-openapi|The serviceName relevant to openAPI definition file. eg. `serviceA` If the serviceName relevant to multiple files should split with `.` eg. `serviceA.API-file1` |The openAPI definitions file contents(yaml format) for create endpoint name grouping rules.|Same as [`productAPI-v2.yaml`](endpoint-grouping-rules.md)|
```

Comment:
> If the serviceName relevant to multiple files should split with `.` eg. `serviceA.API-file1`

This sentence is misleading people, `.` is separating service name and file logic name. Please correct the words.